src/scripts.ts

```diff
@@ -177,7 +181,31 @@ export const createSafeTx = async (provider: string) => {
     }
   }
 
-  const tx = txs.length > 1 ? encodeMulti(txs) : txs[0];
+  let tx: MetaTransaction;
+
+  if (txs.length > 1) {
+    // Ask about zkSync only for multiple transactions
+    const isZkSync = await select({
+      message: "Is your network zkSync?",
+      choices: [
+        { name: "No", value: false },
+        { name: "Yes", value: true },
+      ],
+    });
```

Comment:
This question could be skipped based on the `provider`'s chain id right?

It would be slightly better UX

Reply:
Good shout, I'll fix